### DISCUSSION TITLE
fastify in child process?

### DISCUSSION THREAD

**Discussion by ec2ainun** [fastify/fastify#3353] [General] [answered]
i want to execute a script in background and forget it, using child_process, in that script i want to reuse existing fastify plugin.

my question is.
is it possible to run fastify instance in child_process nodejs? 
if so, can you guys provide some code so that i can tested it?

**Comment by mcollina** [comment, marked as answer]
You can create a fastify server and invoke a route via `.inject()`.

**Reply by ec2ainun** [reply]
thank you, it's works!
